win32ss/user/user32/misc/exticon.c

```diff
@@ -323,6 +323,42 @@ static UINT ICO_ExtractIconExW(
 	}
 	CloseHandle(fmapping);
 
+#ifdef __REACTOS__
+    /* Check if the resource is an animated icon/cursor */
+    if (!memcmp(peimage, "RIFF", 4))
+    {
+        INT     anihOffset;
+        ULONG   uSize = 0;
+
+        /* Get size of the animation data */
+        uSize = MAKEWORD(peimage[4], peimage[5]);
+
+        /* Search for 'anih' indicating animation header */
+        for (anihOffset = 0; anihOffset < uSize; anihOffset++)
```

Comment:
```suggestion
        for (anihOffset = 0; anihOffset + 4 < uSize; anihOffset++)
```

Reply:
why?

Reply:
The below code is checking 4-byte long data...